src/types/RequestOptions.ts

```diff
@@ -70,7 +70,7 @@ export interface RequestOptions {
    *
    * Takes an array of string key/value pairs.
    */
-  customTargeting?: { [key: string]: string };
+  customTargeting?: { [key: string]: string[] };
```

Comment:
When we makes change to type, it is breaking change. To support previous versions, I think it should be union type of `string | string[]`.

Reply:
updated